Libs/Core/ctkAbstractWorker.h

```diff
@@ -48,7 +49,7 @@ class CTK_CORE_EXPORT ctkAbstractWorker : public QObject, public QRunnable
   virtual void run() = 0;
 
   /// Cancel worker
-  virtual void cancel() = 0;
+  virtual void requestCancel() = 0;
```

Comment:
Can this be called from another thread? If yes, add it in a comment. It is important to explicitly state it.

Reply:
yes! ok comment added